WordPress/Docs/WP/CronIntervalStandard.xml

```diff
@@ -0,0 +1,37 @@
+<documentation title="Cron interval">
+    <standard>
+    <![CDATA[
+    Cron schedules running more than once every 15 minutes are discouraged. Crons running more often can negatively impact performance.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Cron schedule is created to run once every hour.">
+        <![CDATA[
+add_filter( 
+    'cron_schedules', 
+    function ( $schedules ) {
+        $schedules['every_hour'] = array(
+            'interval' => HOUR_IN_SECONDS,
+            'display' => __( 'Every hour' )
+        );
+        return $schedules;
+    } 
+); 
+        ]]>
+        </code>
+        <code title="Invalid: Cron schedule is added to run more than once per 15 minutes.">
+        <![CDATA[
+add_filter( 
+    'cron_schedules', 
+    function ( $schedules ) {
+        $schedules['every_9_mins'] = array(
+            'interval' => 9 * 60,
+            'display' => __( 'Every 9 minutes' )
```

Comment:
```suggestion
            'display'  => __( 'Every 9 minutes' )
```

The display name will probably need tweaking as I think it will go over the line length now.